homeassistant/components/google_assistant/trait.py

```diff
@@ -823,14 +878,16 @@ def query_attributes(self):
 
         if domain == cover.DOMAIN:
             return {"isRunning": state in (cover.STATE_CLOSING, cover.STATE_OPENING)}
+        if domain == valve.DOMAIN:
+            return {"isRunning": True}
```

Comment:
We return `True`, so the state is always `running` and the `Stop` function will be always available (if supported)

Reply:
Hm, why don't we rely on CLOSING + OPENING states like we do for cover?

Reply:
I think that should be a UX fix. Replace Start button for open/close depending on the state. And isRunning uses the same logic as the cover.

Reply:
This is not something we can easily customize. It is how the start stop treat works. The integration can decide if a `stop` command should be processed.

Reply:
I suggest I open a separate PR to fix this for cover as well.

Reply:
Another suggestion is to use `Start` to toggle the valve (or cover) thttps://github.com/home-assistant/core/pull/106378